ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>123456789012-12345678901234567890123456789012.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.123456789012-12345678901234567890123456789012</string>
+	<key>API_KEY</key>
+	<string>AasdfDEzghasdfghaEsdfEFghRasdsfghasdfgh</string>
+	<key>GCM_SENDER_ID</key>
+	<string>123456789012</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>org.gooddollar</string>
+	<key>PROJECT_ID</key>
+	<string>good-app</string>
+	<key>STORAGE_BUCKET</key>
+	<string></string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
```

Comment:
is that ok for `.plist` file ?
because in the generic `XML` an single self-closing tag should be used in that case. E.g.:
`<false />`
`<true />`

Reply:
I didn't see any warnings or errors related to this, also, on the firebase dashboard they provide it this way

Reply:
ok